searchguard/rolesmapping.py

```diff
@@ -152,3 +152,22 @@ def modify_rolemapping(role, properties, action="replace"):
 
     # No merge or split action, overwrite existing properties:
     _send_api_request(role, properties)
+
+
+def list_rolemappings_for_user(user, roles):
```

Comment:
Although roles and rolemappings have the same name, isn't it more clear to call this rolemappings?

Reply:
I think that would be more clear, however I wanted to maintain the names of other methods where the parameter for `rolemapping` is referred to as `role`. With this in mind, would you still prefer `rolemappings`? (I'm personally split).

Reply:
Yeah true, let's leave it like this for now.